GoogleVisionBarCodeScanner/Android/Renderer/CameraViewRenderer.cs

```diff
@@ -102,7 +102,15 @@ private void CameraCallback()
                 // Unbind use cases before rebinding
                 cameraProvider.UnbindAll();
                 // Bind use cases to camera
-                _camera = cameraProvider.BindToLifecycle((ILifecycleOwner)Context, cameraSelector, preview, imageAnalyzer);
+                _camera = cameraProvider.BindToLifecycle((ILifecycleOwner)Context, cameraSelector, preview, imageAnalyzer);
+
+                var width = (int?)Element.PreviewWidth.Value;

```

Comment:
I don't think you need to cast to `(int?)` here, since it will throw an exception when accessing `Element.PreviewWidth.Value` if `PreviewWidth` is null. You need to just use `Element.PreviewWidth`